codesydney/taskstech
# Pages consolidate - SignUp & Login

## Rationale

This is for consoliating the pages into one place

## Advice for Reviewers & Testing Notes

- create a new folder src/pages, and import Login(old name TradieLogin) and SignUp(old name TradieSignUp) into Pages.js
- Explain (where relevant) how your change can be tested.
1. create a new user on SignUp page
2. you should be able to login that newly created the user on Login page

## Screenrecords:

- https://user-images.githubusercontent.com/36904351/135746144-85a06c60-60b1-412e-809f-b17d608fb6b2.mov


## Task Name

- Issue #48 


## Linting Checklist
